public/partials/common/navigation.html

```diff
@@ -13,21 +13,21 @@
 
         <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
             <ul class="nav navbar-nav navbar-left">
+                <li class="active">
+                    <a href="/#/dashboard">Dashboard</a>
+                </li>
                 <li>
-                    <a href="">ABIS Home Page <i class="fa fa-external-link fa-fw"></i></a>
```

Comment:
@CamiMonte and @nick22891 wasn't there a requirement for us to be able to link back to the RADA home page from the application?